### ISSUE TITLE
HTML <a  title='_blank'> anchor does not open in a  new window, like other browsers

### ISSUE THREAD

**Issue by icm63** [brave/browser-laptop#13312]
`<a id="myid" title="[CAT] Caterpillar, Inc" class="dt3" href="memSwingChart.aspx?jh65=MjA=&su23=CAT" target="_blank">CAT</a>`

The above URL opens a new window automatically in all other browsers, but not BRAVE, why not ??

I have to select 'open in new window' to get the effect I want .

IS THERE A FIX ??

**Comment by srirambv** [comment]
Does it open in a new tab?

**Comment by ohcibi** [comment]
@srirambv yes it does and so it does in chrome on my computer. IIRC this is configurable in the browsers settings, so I guess OP has different settings in his/her browsers. https://jsfiddle.net/c4w5L9u2/1/

**Comment by icm63** [comment]
Re read my question 

ME: ..."The above URL opens a new window automatically in all other browsers, but not BRAVE, why not ??"...

YES the link open in a new TAB (in brave browser), I dont want that, I want the normal action of a link with a target=_blank to open in a NEW WINDOW (not a new tab)  

NOTE: A new window is a new instance of the browser, a new page  (Not a new tab)

Can BRAVE do this or not ??

Please help??

**Comment by revelt** [comment]
@icm63 I respect your request but let's be objective, it's not "normal action" as you're referring to it, it's an extra feature you are requesting. Personally. I prefer tabs because it's easier to manage tabs than windows. I can see all my tabs while I can't see all my windows.

Should we tag this as a feature request? It's clearly not a bug.

**Comment by bsclifton** [comment]
@revelt good call- let me label the issue appropriately now that I understand the request

**Comment by icm63** [comment]
..."I respect your request but let's be objective, it's not "normal action" as you're referring to it, it's an extra feature you are requesting. Personally. I prefer tabs because it's easier to manage tabs than windows. I can see all my tabs while I can't see all my windows."...

POINT If all other browsers use  <a etc etc target="_blank">LINK</a> to open a new page/window and not a new tab, then you are breaking with HTML convention, which is not smart when trying to win new members, SURE have settings allowing option of NEW WINDOW/PAGE or TAB, but do not turn this standard HTML feature target="_blank"  .... OFF.

**Comment by bsclifton** [comment]
@icm63 you are asking wanting these links to open in a new window, right? As far as I know, all browsers open `_blank` targets in a new tab (well, unless you're using IE6 or another browser which doesn't support tabs)

Since the common behavior is to open in tabs, would you be in favor of an option which allows you to open new tabs in a new window?

**Comment by icm63** [comment]
YOU .."Since the common behavior is to open in tabs, would you be in favor of an option which allows you to open new tabs in a new window?" ...

ME: YES PLEASE

Firefox, Safari, Opera, Chrome, IE, all does

REF: https://www.w3schools.com/tags/att_a_target.asp

_blank | **Opens the linked document in a new window** or tab
_self | Opens the linked document in the same frame as it was clicked (this is default)
_parent | Opens the linked document in the parent frame
_top | Opens the linked document in the full body of the window

**Comment by revelt** [comment]
This could be a preferences switch, definitely. Although I'd keep it off, as it is now :)

**Comment by revelt** [comment]
@icm63 it's off-topic, but you know, you earn trust and respect by a) showing your true identity and b) having a positive track record of open source contributions. Revealing your identity (name etc) shows people you firmly stand by your statements and consider everybody else (same unpaid contributors, just like you) enough not to be sorry of ashamed later. It's very easy to upset people online or to appear inconsiderate. That's why we strive to use politeness online like "please", "could", "would" and positive emoji, also thank people and address any possible misunderstandings upfront. Also, using nameless account which appears _disposable_ devalues the requester's statements to the same level. Just saying.

**Comment by icm63** [comment]
..."it's off-topic, but you know, you earn trust and respect by a) showing your true identity and b) having a positive track record of open source contributions. Revealing your identity (name etc) shows people you firmly stand by your statements and consider everybody else (same unpaid contributors, just like you) enough not to be sorry of ashamed later. It's very easy to upset people online or to appear inconsiderate. That's why we strive to use politeness online like "please", "could", "would" and positive emoji, also thank people and address any possible misunderstandings upfront. Also, using nameless account which appears disposable devalues the requester's statements to the same level. Just saying."...

Ok, maybe, I dont want to help the CIA!

**Comment by icm63** [comment]
I really appreciate this change, tho, thanks!

**Comment by icm63** [comment]
When is it likely to be included in a release ??

**Comment by icm63** [comment]
Looks like this has been done...

**Comment by icm63** [comment]
UPDATE - NOPE , since last update NOT WORKING AGAIN
